src/api_helpers.jl

```diff
@@ -325,6 +325,23 @@ end
 ### Table Interface
 ###
 
+function h5tb_get_table_info(loc_id, table_name)
+    nfields = Ref{hsize_t}()
+    nrecords = Ref{hsize_t}()
+    h5tb_get_table_info(loc_id, table_name, nfields, nrecords)
+    return nfields[], nrecords[]
+end
+
+function h5tb_get_field_info(loc_id, table_name)
+    nfields, = h5tb_get_table_info(loc_id, table_name)
+    field_names = Vector{UInt8}[fill(0x00,255) for i in 1:2]
```

Comment:
I don't know if there is any way to determine the length of these. 
In https://confluence.hdfgroup.org/display/HDF5/H5TB_GET_FIELD_INFO they just allocate 255 sized buffer.